packages/dashboards/addon/routes/dashboards/new.js

```diff
@@ -62,6 +71,7 @@ export default Route.extend({
    * @returns {DS.Model} route model
    */
   _newModel(title = 'Untitled Dashboard') {
+    debugger;
```

Comment:
```suggestion
```

Reply:
Pushed it before you commented, ;) good catch

Reply:
Pushed it before you commented. Good catch :)